LCLSGeneral/LCLSGeneral/POUs/Data/FB_EpicsMotorMonitor.TcPOU

```diff
@@ -0,0 +1,121 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="FB_EpicsMotorMonitor" Id="{c6bf2af8-e9b0-4ebb-b06e-d35a94600f72}" SpecialFunc="None">
+    <Declaration><![CDATA[(*
+    EPICS Motor Record Monitoring tool
```

Comment:
This one I understand, it could get some really effective use with respect to error handling.